adjusted farmerbot with test net details

Mik-TF opened the pull request:
Added farmerbot info for test net

also added scott's video on the farmerbot

updated get started TOC to include new ssh connection section

@xmonader

Mik-TF commented:
@brandonpille 
@scottyeager

Please let me know if something is off. In that case, I can update the file.

I followed your replies (yours and Scott's) [here](https://forum.threefold.io/t/farmerbot-basic-guide-for-qa-net/3845/14).

brandonpille commented on src/farmerbot/farmerbot.md:
```diff
@@ -29,12 +37,12 @@ The key feature of the farmerbot is powermanagement. The farmerbot will automati
 * Set the Farmerbot
 
-Note that this guide is for QA Net. This is a testing environment and farmers do not receive farming rewards (TFT) when deploying on this network. We invite farmers to try this new feature if they want to help test the farmerbot. We will announce when the farmerbot is ready for the Test Net and the Main Net, where farming rewards are distributed.
+The farmerbot is currently available for Test Net and QA Net.
```
Just to be precise for dev net also

Mik-TF replied:
Will change this asap

brandonpille commented on src/farmerbot/farmerbot.md:
```diff
@@ -83,29 +91,19 @@ Your farmerbot can be run on any system, including on a node. It doesn't have to
+To download the Zero-OS bootstrap image, go to the usual bootstrap link [https://v3.bootstrap.grid.tf/](https://v3.bootstrap.grid.tf/) and select the network you want.
 
-![qa_net|690x422](img/farmerbot_4.png) 
+Note that the farmerbot is currently available on QA Net and Test Net.
```
Dev net too.

brandonpille reviewed (approved):
Just one small comment. Already approving.

Mik-TF commented:
@brandonpille 
Done here!

I adjusted the texts in many places to take into account Dev net, as you suggested.

I also added some information and link to the new SSH connection guide. This will help farmers if they want to deploy the farmerbot on a full VM.

I also wrote how much tft is should cost.
